recipes/ninja_syntax/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "ninja_syntax" %}
+{% set version = "1.6.0" %}
+{% set sha256 = "33c203b4ee39a638c9af757ed1c4257e05abf1fae0615305d103020f36c804c5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/43/d2/7a9cafe2b3b6ee260dfeee1a963630c8f0c16620c0ab7d18caab7fb46a42/{{ name }}-{{ version }}.tar.gz
+
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - ninja_syntax
+
+about:
+  home: https://pypi.python.org/pypi/ninja_syntax/1.6.0
```

Comment:
The [ninja_syntax package on pypi](https://pypi.python.org/pypi/ninja_syntax/1.6.0) provides just one file, https://github.com/ninja-build/ninja/blob/master/misc/ninja_syntax.py .

So it's a package for on file of https://github.com/ninja-build/ninja. This file will be duplicated in packages `ninja` and `ninja_syntax`.

`ninja_syntax` allows python modules to generate .ninja files, without installing the entire `ninja` (which imply compiling C++).

For a conda point of view, one package would be enough, but software such as [craftr](https://github.com/craftr-build/craftr) do [import](https://github.com/craftr-build/craftr/blob/cc94a5440eb3d7b31ab8e5f516a74880f040a309/craftr/core/build.py#L32) `ninja_syntax`.